requirements.txt

```diff
@@ -3,6 +3,7 @@
 click>=8.1.7,<9.0.0
 click-didyoumean>=0.3.0
 datasets>=2.18.0
+docling-parse>=2.0.0,<3.0.0
```

Comment:
We have some customizations in the build downstream for docling-parse, but they don't look like anything that would be affected by this and we're currently building version 2.1.2. If 3.0 _is_ significantly breaking, we'll want to coordinate the update so the builder code is ready when upstream switches this to `>=3.0.0`.

Reply:
Removed and introduced in the `instructlab-sdg` package. See SDG release [v0.6.2](https://github.com/instructlab/sdg/releases/tag/v0.6.2).